src/openads/application/OpenAds.js

```diff
@@ -86,4 +86,13 @@ export default class OpenAds {
   environment () {
     return this._container.config
   }
+
+  /**
+   * Displays a position in the page
+   * @param {string} position id
+   * @return {Promise<Position>}
+   */
+  displayPosition ({id} = {}) {
```

Comment:
why this method has ({id} = {}) ?

Reply:
To set an initial value in order to avoid dealing with undefined when calling this method without parameters.